tensorflow_datasets/core/dataset_builder.py

```diff
@@ -319,6 +319,46 @@ def as_dataset(self,
     """Constructs a `tf.data.Dataset`.
 
     Callers must pass arguments as keyword arguments.
+    
+    The output types vary depending on the parameters. Examples:
+    
+    ```python
+    ds_builder = tfds.text.imdb.IMDBReviews()
+    ds_builder.download_and_prepare()
+    
+    # Default parameters
+    ds1 = ds_builder.as_dataset()
```

Comment:
Yes it's a small script example but the names of `ds1` and `ds2` may be more clearer and understandable.